apps/tlon-web/src/state/channel/channel.ts

```diff
@@ -1139,6 +1135,7 @@ export function useChannelsFirehose() {
   }, []);
 
   const eventHandler = useCallback((event: ChannelsSubscribeResponse) => {
+    console.log('received channel update', event);
```

Comment:
```suggestion
```

Do we want all of these console logs all the time in this file? We could use createDevLogger.